src/main/java/Duke.java

```diff
@@ -1,10 +1,346 @@
+
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import duke.task.Todo;
+import duke.task.Task;
+import duke.task.TestTask;
+import duke.task.Event;
+import duke.task.Deadline;
+import duke.exception.DukeException;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
+    static int listLength;
+
+    static Task task;
+
+
+    static String userInput;
+    static String command;
+    static String description = "No description is needed for this command";
+    public static final String SEPARATOR = "-----------------------------------";
+    private static ArrayList<Task> tasks;
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
+
         System.out.println("Hello from\n" + logo);
+
+
+
+
+
+        setTasks();
+
+
+        Scanner in = new Scanner(System.in);
+        userInput = in.nextLine();
+
+        while (!userInput.equals("bye")) {
+            try {
+                if (userInput.equals("echo")) {
```

Comment:
There are several `if` conditions that you have specified. Some are explicit enough like  `echo` and `list`, but there are also varying input lengths to check. 

1.  You could have a separate parser method to determine what type of input it is.
3. Each input type can be handled separately as cases if there are too many to handle. Refer to the [standard switch-case layout](https://se-education.org/guides/conventions/java/basic.html#statements).